### DISCUSSION TITLE
How do I disable monitoring RDP?

### DISCUSSION THREAD

**Discussion by ACSMid** [DigitalRuby/IPBan#301] [Q&A] [answered]
I'm thrilled to have found IPBan. My MailEnable server is under constant bombardment. The biggest issue I'm having is false positives coming from cell phones. This results in calls of users complaining their email doesn't work. RDP always seems to be the culperate. I want to disable IPBan from monitoring RDP to get around this. I tried commenting out the RDP parts in the config, but I most not have done it correclty. I don't see a simple toggle in the config either.

**Comment by jjxtra** [comment, marked as answer]
There's four `<Source>RDP</Source>` elements in the ipban.config. Comment all four containing `<Group>` elements out.
